### PR TITLE
Project not found instead of not specified

### PR DESCRIPTION
**Pull request by vboctor**
If project id or name is specified but not found, show proper error message.

Fixes #25360

### CODE REVIEW

File: core/commands/IssueAddCommand.php [hunk 1]
```diff
@@ -135,16 +135,27 @@ function validate() {
...
+			!isset( $t_issue['project']['id'] ) && !isset( $t_issue['project']['name'] ) ) {
 			throw new ClientException(
 				'Project not specified',
 				ERROR_EMPTY_FIELD,
 				array( 'project' ) );
 		}
 
 		if( !project_exists( $t_project_id ) ) {
+			if( $t_project_id == ALL_PROJECTS ) {
+				if( isset( $t_issue['project']['id'] ) && !is_blank( isset( $t_issue['project']['id'] ) ) ) {
```
Comment (atrol) [inline core/commands/IssueAddCommand.php]: I don't understand what you want with`!is_blank( isset ...`
Test if a bolean is not blank?

File: core/commands/IssueAddCommand.php [hunk 2]
```diff
@@ -135,16 +135,27 @@ function validate() {
...
+					$t_project_id = isset( $t_issue['project']['id'] );
```
Comment (atrol) [inline core/commands/IssueAddCommand.php]: What's the purpose of assigning a boolean to a project id?

File: core/commands/IssueAddCommand.php [hunk 3]
```diff
@@ -135,16 +135,27 @@ function validate() {
...
+				} else if( isset( $t_issue['project']['name'] ) && !is_blank( isset( $t_issue['project']['name'] ) ) ) {
+					$t_project_id = $t_issue['project']['name'];
+				} else {
+					$t_project_id = ALL_PROJECTS;
```
Comment (atrol) [inline core/commands/IssueAddCommand.php]: No need to assign it again as the code block starts with `if( $t_project_id == ALL_PROJECTS ) {`

**Comment by dregad** [comment]
@vboctor are you going to address atrol's review comments ?

**Comment by vboctor** [comment]
Closing this in favor of https://github.com/mantisbt/mantisbt/pull/1873

**Pull request closed without merging**